javascript/javascript1/week4/yasi/codewars/ square-every-digit.js

```diff
@@ -0,0 +1,3 @@
+function squareDigits(num){
+    return Number(num.toString().split('').map(digit => digit ** 2).join(''));
+  }
```

Comment:
Be careful with indentation: the `}` closing a function should be aligned with the `function` keyword like this
```js
function squareDigits(num){
    return Number(num.toString().split('').map(digit => digit ** 2).join(''));
}
```

This comment is valid for the other files too.